src/alchemtest/tests/__init__.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+class BaseDatasetTest:
+    # @pytest.fixture(scope="class",
+    #                 params = [(load_tyr2ala, ('forward', 'backward'), (1, 1)),
+    #                           ])
+    # def dataset(self, request):
+    #     '''The input dataset is specified as:
+    #     load_method: The method for loading the dataset.
+    #     keys: The keys to the corresponding dataset.
+    #     value_length: The expected number of file for each key.
+    #     '''
+    #     load_method, keys, value_length = request.param
+    #     return load_method, keys, value_length
```

Comment:
Any comments how we could keep this code in the base class while still using class-local fixtures would be welcome.